view/frontend/templates/page/analytics.phtml

```diff
@@ -2,11 +2,12 @@
 /**
  * @see Metrilo\Analytics\Block\Analytics
  */
-$event = $block->getEvent();
-$url   = $block->getLibraryUrl();
+
+$url             = $block->getLibraryUrl();
+$addToCartEvents = $block->getAddToCartEvents();
 ?>
 <script type="text/javascript">
     require(['<?php echo $url; ?>'], function(){
-        <?php if($event){ echo $event; } ?>
+        <?php echo $block->getEvents(); ?>
```

Comment:
you should probably just iterate over the `getEvents` array here and print with echo